Doc/c-api/intro.rst

```diff
@@ -174,6 +174,17 @@ complete listing.
    .. versionchanged:: 3.8
       MSVC support was added.
 
+.. c:macro:: PyDoc_STRVAR(name, str)
+
+   A shortcut for ``static const char name[] = str``, commonly used
+   to initialize a module, function, class, or method docstring.
+
+   Example::
+
+      PyDoc_STRVAR(module_doc, "This is the module's docstring.");
```

Comment:
I think expanding this to:

```c
PyDoc_STRVAR(method_doc, "This is the method's docstring.");
// ...
    {"my_method", my_method, METH_O, method_doc},
```

would be useful to actually show a place where the macro-generated variable is used.

Reply:
I used a method docstring directly from `_collectionsmodule.c`